invenio_accounts/ext.py

```diff
@@ -205,6 +202,19 @@ def delay_security_email(msg):
         if app.config.get('ACCOUNTS_USERINFO_HEADERS'):
             request_finished.connect(set_session_info, app)
 
+        # Set Session KV store
+        if app.config.get('ACCOUNTS_SESSION_STORE_FACTORY'):
+            factory = load_obj_or_import_string(
+                app.config.get('ACCOUNTS_SESSION_STORE_FACTORY'))
+            session_kvstore = factory(app)
+        else:
+            from simplekv.memory import DictStore
```

Comment:
This block could also be provided to the `ACCOUNTS_SESSION_STORE_FACTORY ` as the default factory.